.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,86 @@
+# As much as possible, this file should be kept in sync with
+# https://github.com/napari/docs/blob/main/.github/workflows/build_and_deploy_docs.yml
+name: Build PR Docs
+
+on:
+  pull_request:
+    types:
+        - opened
+        - synchronize
```

Comment:
We'll want to replace this with:
```
branches:
      - main
```
but then it wouldn't run in my fork for demo purposes.

Reply:
done

Reply:
Done.